From bf79bdebf45bbd78ad4f456817a94601aa6415f0 Mon Sep 17 00:00:00 2001
From: Yonghua Huang <yonghua.huang@intel.com>
Date: Thu, 22 Feb 2018 23:13:21 +0800
Subject: [PATCH 1117/2345] [sos] rename 'cwp_*' header file in VHM

  - cwp_vhm_ioreq.h --> acrn_vhm_ioreq.h
  - cwp_hv_defs.h   --> acrn_hv_defs.h
  - cwp_common.h    --> acrn_common.h
  - cwp_vhm_mm.h    --> acrn_vhm_mm.h

Change-Id: Ic83bd7ddb308658eeb6570ebc26a59d8c564baaf
Signed-off-by: Yonghua Huang <yonghua.huang@intel.com>
---
 Documentation/virtual/cwp/vhm.rst                       | 4 ++--
 drivers/char/vhm/vhm_dev.c                              | 6 +++---
 drivers/cwp/sbuf.c                                      | 2 +-
 drivers/gpu/drm/i915/gvt/cwpgt.c                        | 8 ++++----
 drivers/vbs/vq.c                                        | 2 +-
 drivers/vhm/vhm_hypercall.c                             | 2 +-
 drivers/vhm/vhm_ioreq.c                                 | 4 ++--
 drivers/vhm/vhm_mm.c                                    | 4 ++--
 drivers/vhm/vhm_msi.c                                   | 2 +-
 drivers/vhm/vhm_vm_mngt.c                               | 6 +++---
 include/linux/vbs/vbs.h                                 | 4 ++--
 include/linux/vhm/{cwp_common.h => acrn_common.h}       | 0
 include/linux/vhm/{cwp_hv_defs.h => acrn_hv_defs.h}     | 2 +-
 include/linux/vhm/{cwp_vhm_ioreq.h => acrn_vhm_ioreq.h} | 2 +-
 include/linux/vhm/{cwp_vhm_mm.h => acrn_vhm_mm.h}       | 2 +-
 include/linux/vhm/vhm_ioctl_defs.h                      | 2 +-
 16 files changed, 26 insertions(+), 26 deletions(-)
 rename include/linux/vhm/{cwp_common.h => acrn_common.h} (100%)
 rename include/linux/vhm/{cwp_hv_defs.h => acrn_hv_defs.h} (99%)
 rename include/linux/vhm/{cwp_vhm_ioreq.h => acrn_vhm_ioreq.h} (99%)
 rename include/linux/vhm/{cwp_vhm_mm.h => acrn_vhm_mm.h} (99%)

diff --git a/Documentation/virtual/cwp/vhm.rst b/Documentation/virtual/cwp/vhm.rst
index 215947c..a201fc7 100644
--- a/Documentation/virtual/cwp/vhm.rst
+++ b/Documentation/virtual/cwp/vhm.rst
@@ -8,7 +8,7 @@ Clearwater Pass (CWP) Project.
 APIs:
 -----
 
-.. kernel-doc:: include/linux/vhm/cwp_vhm_ioreq.h
-.. kernel-doc:: include/linux/vhm/cwp_vhm_mm.h
+.. kernel-doc:: include/linux/vhm/acrn_vhm_ioreq.h
+.. kernel-doc:: include/linux/vhm/acrn_vhm_mm.h
 .. kernel-doc:: include/linux/vhm/vhm_ioctl_defs.h
 .. kernel-doc:: include/linux/vhm/vhm_vm_mngt.h
diff --git a/drivers/char/vhm/vhm_dev.c b/drivers/char/vhm/vhm_dev.c
index 1bdde44..86f1afb 100644
--- a/drivers/char/vhm/vhm_dev.c
+++ b/drivers/char/vhm/vhm_dev.c
@@ -77,10 +77,10 @@
 #include <linux/pci.h>
 #include <linux/list.h>
 
-#include <linux/vhm/cwp_hv_defs.h>
+#include <linux/vhm/acrn_hv_defs.h>
 #include <linux/vhm/vhm_ioctl_defs.h>
-#include <linux/vhm/cwp_vhm_ioreq.h>
-#include <linux/vhm/cwp_vhm_mm.h>
+#include <linux/vhm/acrn_vhm_ioreq.h>
+#include <linux/vhm/acrn_vhm_mm.h>
 #include <linux/vhm/vhm_vm_mngt.h>
 #include <linux/vhm/vhm_hypercall.h>
 
diff --git a/drivers/cwp/sbuf.c b/drivers/cwp/sbuf.c
index cd38248..a358232 100644
--- a/drivers/cwp/sbuf.c
+++ b/drivers/cwp/sbuf.c
@@ -57,7 +57,7 @@
 
 #include <linux/gfp.h>
 #include <asm/pgtable.h>
-#include <linux/vhm/cwp_hv_defs.h>
+#include <linux/vhm/acrn_hv_defs.h>
 #include <linux/vhm/vhm_hypercall.h>
 #include "sbuf.h"
 
diff --git a/drivers/gpu/drm/i915/gvt/cwpgt.c b/drivers/gpu/drm/i915/gvt/cwpgt.c
index 490e0f8..f3b96f4 100644
--- a/drivers/gpu/drm/i915/gvt/cwpgt.c
+++ b/drivers/gpu/drm/i915/gvt/cwpgt.c
@@ -34,10 +34,10 @@
 #include <linux/wait.h>
 #include <linux/sched.h>
 
-#include <linux/vhm/cwp_hv_defs.h>
-#include <linux/vhm/cwp_common.h>
-#include <linux/vhm/cwp_vhm_ioreq.h>
-#include <linux/vhm/cwp_vhm_mm.h>
+#include <linux/vhm/acrn_hv_defs.h>
+#include <linux/vhm/acrn_common.h>
+#include <linux/vhm/acrn_vhm_ioreq.h>
+#include <linux/vhm/acrn_vhm_mm.h>
 #include <linux/vhm/vhm_vm_mngt.h>
 
 #include <i915_drv.h>
diff --git a/drivers/vbs/vq.c b/drivers/vbs/vq.c
index 635c4bd..e8435a1 100644
--- a/drivers/vbs/vq.c
+++ b/drivers/vbs/vq.c
@@ -63,7 +63,7 @@
 #include <linux/module.h>
 #include <linux/vbs/vq.h>
 #include <linux/vbs/vbs.h>
-#include <linux/vhm/cwp_vhm_mm.h>
+#include <linux/vhm/acrn_vhm_mm.h>
 
 /* helper function for remote memory map */
 void * paddr_guest2host(struct ctx *ctx, uintptr_t gaddr, size_t len)
diff --git a/drivers/vhm/vhm_hypercall.c b/drivers/vhm/vhm_hypercall.c
index 235c6b2..44fd8a2 100644
--- a/drivers/vhm/vhm_hypercall.c
+++ b/drivers/vhm/vhm_hypercall.c
@@ -49,7 +49,7 @@
  *
  */
 #include <linux/types.h>
-#include <linux/vhm/cwp_hv_defs.h>
+#include <linux/vhm/acrn_hv_defs.h>
 #include <linux/vhm/vhm_hypercall.h>
 
 inline long hcall_get_api_version(unsigned long api_version)
diff --git a/drivers/vhm/vhm_ioreq.c b/drivers/vhm/vhm_ioreq.c
index 74427bc..5c5a00a 100644
--- a/drivers/vhm/vhm_ioreq.c
+++ b/drivers/vhm/vhm_ioreq.c
@@ -60,8 +60,8 @@
 #include <linux/mm.h>
 #include <linux/poll.h>
 #include <linux/delay.h>
-#include <linux/vhm/cwp_common.h>
-#include <linux/vhm/cwp_vhm_ioreq.h>
+#include <linux/vhm/acrn_common.h>
+#include <linux/vhm/acrn_vhm_ioreq.h>
 #include <linux/vhm/vhm_vm_mngt.h>
 #include <linux/vhm/vhm_hypercall.h>
 
diff --git a/drivers/vhm/vhm_mm.c b/drivers/vhm/vhm_mm.c
index 7a3978d..e35b7b2 100644
--- a/drivers/vhm/vhm_mm.c
+++ b/drivers/vhm/vhm_mm.c
@@ -71,8 +71,8 @@
 #include <linux/io.h>
 
 #include <linux/vhm/vhm_ioctl_defs.h>
-#include <linux/vhm/cwp_hv_defs.h>
-#include <linux/vhm/cwp_vhm_mm.h>
+#include <linux/vhm/acrn_hv_defs.h>
+#include <linux/vhm/acrn_vhm_mm.h>
 #include <linux/vhm/vhm_vm_mngt.h>
 #include <linux/vhm/vhm_hypercall.h>
 
diff --git a/drivers/vhm/vhm_msi.c b/drivers/vhm/vhm_msi.c
index 05e166b..83db860 100644
--- a/drivers/vhm/vhm_msi.c
+++ b/drivers/vhm/vhm_msi.c
@@ -53,7 +53,7 @@
 
 #include <linux/msi.h>
 #include <linux/pci.h>
-#include <linux/vhm/cwp_hv_defs.h>
+#include <linux/vhm/acrn_hv_defs.h>
 #include <linux/vhm/vhm_hypercall.h>
 
 #include "../pci/pci.h"
diff --git a/drivers/vhm/vhm_vm_mngt.c b/drivers/vhm/vhm_vm_mngt.c
index 100435d..ea0be4b 100644
--- a/drivers/vhm/vhm_vm_mngt.c
+++ b/drivers/vhm/vhm_vm_mngt.c
@@ -56,10 +56,10 @@
 #include <linux/slab.h>
 #include <linux/init.h>
 #include <asm/processor.h>
-#include <linux/vhm/cwp_hv_defs.h>
+#include <linux/vhm/acrn_hv_defs.h>
 #include <linux/vhm/vhm_ioctl_defs.h>
-#include <linux/vhm/cwp_vhm_ioreq.h>
-#include <linux/vhm/cwp_vhm_mm.h>
+#include <linux/vhm/acrn_vhm_ioreq.h>
+#include <linux/vhm/acrn_vhm_mm.h>
 #include <linux/vhm/vhm_hypercall.h>
 
 LIST_HEAD(vhm_vm_list);
diff --git a/include/linux/vbs/vbs.h b/include/linux/vbs/vbs.h
index 415d637..58457d6 100644
--- a/include/linux/vbs/vbs.h
+++ b/include/linux/vbs/vbs.h
@@ -63,8 +63,8 @@
 #define _VBS_H_
 
 #include <linux/vbs/vbs_common_if.h>
-#include <linux/vhm/cwp_common.h>
-#include <linux/vhm/cwp_vhm_ioreq.h>
+#include <linux/vhm/acrn_common.h>
+#include <linux/vhm/acrn_vhm_ioreq.h>
 
 /**
  * enum IORangeType - type of registers to be handled in VBS-K
diff --git a/include/linux/vhm/cwp_common.h b/include/linux/vhm/acrn_common.h
similarity index 100%
rename from include/linux/vhm/cwp_common.h
rename to include/linux/vhm/acrn_common.h
diff --git a/include/linux/vhm/cwp_hv_defs.h b/include/linux/vhm/acrn_hv_defs.h
similarity index 99%
rename from include/linux/vhm/cwp_hv_defs.h
rename to include/linux/vhm/acrn_hv_defs.h
index 52f164c..636922d 100644
--- a/include/linux/vhm/cwp_hv_defs.h
+++ b/include/linux/vhm/acrn_hv_defs.h
@@ -55,7 +55,7 @@
 /*
  * Common structures for CWP/VHM/DM
  */
-#include "cwp_common.h"
+#include "acrn_common.h"
 
 /*
  * Common structures for HV/VHM
diff --git a/include/linux/vhm/cwp_vhm_ioreq.h b/include/linux/vhm/acrn_vhm_ioreq.h
similarity index 99%
rename from include/linux/vhm/cwp_vhm_ioreq.h
rename to include/linux/vhm/acrn_vhm_ioreq.h
index 1f24e24..a159ab9 100644
--- a/include/linux/vhm/cwp_vhm_ioreq.h
+++ b/include/linux/vhm/acrn_vhm_ioreq.h
@@ -52,7 +52,7 @@
  */
 
 /**
- * @file cwp_vhm_ioreq.h
+ * @file acrn_vhm_ioreq.h
  *
  * @brief Virtio and Hypervisor Module(VHM) ioreq APIs
  */
diff --git a/include/linux/vhm/cwp_vhm_mm.h b/include/linux/vhm/acrn_vhm_mm.h
similarity index 99%
rename from include/linux/vhm/cwp_vhm_mm.h
rename to include/linux/vhm/acrn_vhm_mm.h
index c22dc29..d3d17ce 100644
--- a/include/linux/vhm/cwp_vhm_mm.h
+++ b/include/linux/vhm/acrn_vhm_mm.h
@@ -52,7 +52,7 @@
  */
 
 /**
- * @file cwp_vhm_mm.h
+ * @file acrn_vhm_mm.h
  *
  * @brief Virtio and Hypervisor Module memory manager APIs
  */
diff --git a/include/linux/vhm/vhm_ioctl_defs.h b/include/linux/vhm/vhm_ioctl_defs.h
index 64c35d8..c10ff99 100644
--- a/include/linux/vhm/vhm_ioctl_defs.h
+++ b/include/linux/vhm/vhm_ioctl_defs.h
@@ -53,7 +53,7 @@
 #define	_VHM_IOCTL_DEFS_H_
 
 /* Commmon structures for CWP/VHM/DM */
-#include "cwp_common.h"
+#include "acrn_common.h"
 
 /*
  * Commmon IOCTL ID defination for VHM/DM
-- 
2.7.4

